Tools/scripts/generate_global_objects.py

```diff
@@ -105,6 +105,7 @@ def iter_files():
 
 
 def iter_global_strings():
+    # Use of _Py_ID() is ignored in pycore_runtime_init_generated.h.
```

Comment:
The comment helps.  However, I recommend moving it down to the generated `_Py_ID()` in generate_runtime_init(), along with a slight adjustment.
```suggestion
```

Reply:
done